lua/illuminate/config.lua

```diff
@@ -12,6 +12,14 @@ local config = {
         'dirbuf',
         'dirvish',
         'fugitive',
+        "startify",
+        "packer",
+        "NvimTree",
+        "Trouble",
+        "dashboard",
+        "alpha",
+        "TelescopePrompt",
+        "TelescopeResults",
```

Comment:
Use single quotes. Also include a reference/description for each filetype, I'm not sure what all of these are.

Reply:
Also if you can sort the list alphabetically.